README.md

```diff
@@ -69,6 +69,56 @@ you're ready, switch to using the `package.json` `"scripts"` entry instead.)
 
 ### Schedule a job:
 
+There are two ways to schedule jobs:
+
+1. From Node with the `WorkerUtils` or `addJob` APIs.
+2. In the database directly with the `add_job` SQL function.
+
+#### The WorkerUtils API
+
+Import `WorkerUtils` from `graphile_worker`, and make a new instance by passing in your database configuration options (just like you would with `run` above). What you get back is an instance of a class with a function called `addJob`, and a function called `end`.
+
+The `addJob` function takes a name, and a payload, and returns a promise of a job that has been added to the queue. The `end` function closes the connection, don't forget to call it yourself if you're done adding jobs and don't want to keep the pool open anymore!
+
+```js
+const { WorkerUtils } = require("graphile-worker");
+
+async function main() {
+  const workerUtils = new WorkerUtils({ connectionString: "postgres:///" });
+  try {
+    await workerUtils.addJob("calculate-life-meaning", { value: 42 });
+  } finally {
+    workerUtils.end();
+  }
+}
+
+main().catch(err => {
+  console.error(err);
+  process.exit(1);
+});
+```
+
+##### The `addJob` standalone Javascript function
+
+If you just want the fast and easy way to add a job, and you don't mind the cost of opening a DB connection pool, and then cleaning it up right away _for every job added_, there's the `addJob` convenience function:
```

Comment:
```suggestion
If you just want the fast and easy way to add a job, and you don't mind the cost of opening a DB connection pool, and then cleaning it up right away _for every job added_, there's the `addJob` convenience function. It takes the `WorkerUtils` configuration as the first argument, and then the remaining `WorkerUtils#addJob` arguments afterwards:
```

Reply:
Note: I'm trying to be careful to use "configuration" for the WorkerUtils argument, and "options" for addJob to avoid confusion of just using "options" everywhere.